moviepy/video/io/ffmpeg_tools.py

```diff
@@ -66,4 +66,30 @@ def ffmpeg_resize(video,output,size):
              output]
              
     subprocess_call(cmd)
+    
+def ffmpeg_stabilize_video(filename, output = None):
+    """
+
+    Makes a new video file, from the given one
+    which is more stable
+
+    Parameters
+    -----------
+
+    filename
+    The name of the shaky video
+
+    output
+    The name of new stabilazed video
+    """
+
+    name, ext = os.path.splitext(filename)
+    if not output:
+        output = "%s_stabilazed%s"%(name, ext)
```

Comment:
Can you please rename `stabilazed` to `stabilized` here.  I was going to say, we might want to elect to use the new style string formatting as well `output = '{0}_stabilized{1}.format(name, ext)`, but it seems we're using the old style throughout the file, so: `output = %s_stabilized%s" % (name, ext)` should be fine.